admin/views/school/save-pupils-not-taking-check.ejs

```diff
@@ -0,0 +1,67 @@
+<div class="grid-row">
+    <div class="column-one-half">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-half">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row" id="pupils-not-taking-checks" aria-describedby="heading-xlarge">
+
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge" id="heading-xlarge" tabindex="0">Pupils not taking the check</h1>
```

Comment:
Have you a link to any research that suggests adding a tab index is useful for `h` elements?  It is not included on the gov.uk documentation: https://govuk-elements.herokuapp.com/typography/#typography-headings for instance.

Reply:
I've removed it for now but I'll bring this back eventually when we revise accessibility. The purpose is to improve the keyboard navigation.